include/zephyr/bluetooth/mesh/brg_cfg_cli.h

```diff
@@ -0,0 +1,315 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_BLUETOOTH_MESH_BRG_CFG_CLI_H__
+#define ZEPHYR_INCLUDE_BLUETOOTH_MESH_BRG_CFG_CLI_H__
+
+#include <zephyr/bluetooth/mesh.h>
+#include <zephyr/bluetooth/mesh/brg_cfg.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @defgroup bt_mesh_brg_cfg_cli Bridge Configuration Client Model
+ * @ingroup bt_mesh
+ * @{
+ *  @brief API for the Bluetooth Mesh Bridge Configuration Client model
```

Comment:
```suggestion
 *  @brief API for the Bluetooth Mesh Subnetwork Bridge Configuration Client model
```

Reply:
Will keep the name from the Mesh Protocol spec.